lib/sgov/entity-adapters/sgov-entity-adapter.ts

```diff
@@ -0,0 +1,30 @@
+import {RdfObject, RdfSourceWrap} from "../../core/adapter/rdf";
+import {PimResource} from "../../pim/model";
+import {LanguageString} from "../../core";
+import {SKOS} from "../sgov-vocabulary";
+import {IriProvider} from "../../cim/iri-provider";
+
+// todo move
```

Comment:
There should be no TOODs in the code, it is better to use issues - but t be honest I have TODOs as well. So instead please clarify, like move what .. move where .. why?